classes/table/history.php

```diff
@@ -118,8 +118,16 @@ public function download() {
      * @return string
      */
     public function col_duration($values) {
-        $lockcount = $values->lockcount;
-        $duration = $values->duration;
+        $lockcount = null;
+        $duration = null;
+
+        if (isset($values->lockcount)) {
+            $lockcount = $values->lockcount;
+        }
+
+        if (isset($values->duration)) {
+            $duration = $values->duration;
+        }
```

Comment:
You could make this slightly more succinct by using a ternary operator, like:

$lockcount = isset($values->lockcount) ? $values->lockcount : null;

A lot of people intensely dislike ternary operators, but I think they're handy for these kinds of simple checks.